### PR TITLE
Feature/pim 1760 verified yield grain rollover backend

### PR DESCRIPTION
**Pull request by dpitk2222**

### CODE REVIEW

File: cirras-underwriting-api/cirras-underwriting-api-rest-endpoints/src/main/java/ca/bc/gov/mal/cirras/underwriting/api/rest/v1/resource/factory/UwContractRsrcFactory.java [hunk 1]
```diff
@@ -201,6 +202,7 @@ private void populateResource(UwContractRsrc resource, PolicyDto dto) {
 		resource.setGrowerPrimaryPhone(dto.getGrowerPrimaryPhone());
 		resource.setInventoryContractGuid(dto.getInventoryContractGuid());
 		resource.setDeclaredYieldContractGuid(dto.getDeclaredYieldContractGuid());
+		resource.setVerifiedYieldContractGuid(null); // TODO: Set from dto.
```
Comment (mhostett) [inline cirras-underwriting-api/cirras-underwriting-api-rest-endpoints/src/main/java/ca/bc/gov/mal/cirras/underwriting/api/rest/v1/resource/factory/UwContractRsrcFactory.java:205]: Is this done in PIM-1717?

File: cirras-underwriting-api/cirras-underwriting-model/src/main/java/ca/bc/gov/mal/cirras/underwriting/model/v1/VerifiedYieldContract.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package ca.bc.gov.mal.cirras.underwriting.model.v1;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+public interface VerifiedYieldContract<A extends AnnualField> extends Serializable {
```
Comment (mhostett) [inline cirras-underwriting-api/cirras-underwriting-model/src/main/java/ca/bc/gov/mal/cirras/underwriting/model/v1/VerifiedYieldContract.java:7]: Is _extends AnnualField_ necessary?

**Review by mhostett — CHANGES_REQUESTED** [review]

File: cirras-underwriting-api/cirras-underwriting-api-rest-endpoints/src/main/java/ca/bc/gov/mal/cirras/underwriting/api/rest/v1/resource/factory/UwContractRsrcFactory.java (hunk 1 above)
Comment (dpitk2222) [reply cirras-underwriting-api/cirras-underwriting-api-rest-endpoints/src/main/java/ca/bc/gov/mal/cirras/underwriting/api/rest/v1/resource/factory/UwContractRsrcFactory.java:205]: Yes, that's what I was thinking.

File: cirras-underwriting-api/cirras-underwriting-model/src/main/java/ca/bc/gov/mal/cirras/underwriting/model/v1/VerifiedYieldContract.java (hunk 2 above)
Comment (dpitk2222) [reply cirras-underwriting-api/cirras-underwriting-model/src/main/java/ca/bc/gov/mal/cirras/underwriting/model/v1/VerifiedYieldContract.java:7]: I wasn't sure. It depends on the questions we asked in PIM-1699. We can either included a "fields" member like we do for InventoryContract and DopContract, in which case we do need this, or use an endpoint to load the fields for the contract separately, in which case we probably don't need this.

Comment (mhostett) [reply cirras-underwriting-api/cirras-underwriting-model/src/main/java/ca/bc/gov/mal/cirras/underwriting/model/v1/VerifiedYieldContract.java:7]: Ok, let's leave it in until we have all the answers we need.

**Review by mhostett — APPROVED** [review]

**Pull request merged**